### PR TITLE
fix(build): Fix make dev issue

### PR DESCRIPTION
**Pull request by jim-wang-intel**
Added back the overridden env vars for gen target: CORE_EDGEX_REPOSITORY and CORE_EDGEX_VERSION

Signed-off-by: Jim Wang <yutsung.jim.wang@intel.com>

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/developer-scripts/blob/master/.github/Contributing.md.

## What is the current behavior?

`make run/gen dev` generates the compose file containing non-existing docker images for developers

## What is the new behavior?
fix the above-mentioned issue.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x ] No

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?

## Other information

### CODE REVIEW

File: compose-builder/Makefile [hunk 1]
```diff
@@ -256,6 +256,8 @@ gen:
 	$(GROVE_CHECK) \
 	DEV=$(DEV) \
 	APP_SVC_DEV=$(APP_SVC_DEV) \
+	CORE_EDGEX_REPOSITORY=$(CORE_EDGEX_REPOSITORY) \
+	CORE_EDGEX_VERSION=$(CORE_EDGEX_VERSION) \
```
Comment (lenny-intel) [inline compose-builder/Makefile:260]: Also need `APP_SVC_REPOSITORY` and `APP_SERVICE_VERSION` put back

**Review by lenny-intel — CHANGES_REQUESTED** [review]
Please also remove the echo of OPTIONS from the help target.

File: compose-builder/Makefile (hunk 1 above)
Comment (jim-wang-intel) [reply compose-builder/Makefile:260]: good catch. forget about these

Comment (jim-wang-intel) [reply compose-builder/Makefile:260]: actually there is no APP_SERVICE_VERSION to be overridden

Comment (jim-wang-intel) [reply compose-builder/Makefile:260]: fixed

**Comment by jim-wang-intel** [comment]
> Please also remove the echo of OPTIONS from the help target.

Removed. Please review it again. Thanks.

**Review by lenny-intel — APPROVED** [review]

**Comment by lenny-intel** [comment]
recheck

**Pull request merged**